gtk4/src/subclass/check_button.rs

```diff
@@ -11,10 +11,15 @@ pub trait CheckButtonImpl: CheckButtonImplExt + WidgetImpl {
     fn toggled(&self, check_button: &Self::Type) {
         self.parent_toggled(check_button)
     }
+
+    fn activate(&self, check_button: &Self::Type) {
```

Comment:
This one is problematic probably no? Not sure if we can do much about it.

Reply:
Problematic in what sense?

Reply:
it's a new virtual method that got added in v4_2. It's already marked with `Since` but that doesn't show up in `GtkCheckButtonClass`

Reply:
That's ok, you'd put the `#[cfg]` yourself into the `subclass/check_button.rs`. It doesn't matter for -sys here.